apps/opik-backend/src/main/java/com/comet/opik/api/Project.java

```diff
@@ -29,7 +32,15 @@ public record Project(
         @JsonView({Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) Instant lastUpdatedAt,
         @JsonView({Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) String lastUpdatedBy,
         @JsonView({
-                Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) @Nullable Instant lastUpdatedTraceAt){
+                Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) @Nullable Instant lastUpdatedTraceAt,
+        @JsonView({
+                Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) @Nullable List<FeedbackScoreAverage> feedbackScores,
+        @JsonView({
+                Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) @Nullable PercentageValues duration,
+        @JsonView({
+                Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) @Nullable Double totalEstimatedCost,
```

Comment:
Monetary cost as floating type. For other aggregations such as avg, it wouldn't be that bad. But for a total, I assume is a sum, better as BigDecimal.

Reply:
The name is a bit misleading. This holds the average total cost of a trace in a project. This is calculated here:
https://github.com/comet-ml/opik/blob/ab0f8bb5074e4a12bde2b6a5af5ad161d28f1088/apps/opik-backend/src/main/java/com/comet/opik/domain/TraceDAO.java#L615

In addition, you can see that the coverage for this logic asserts on an average:
https://github.com/comet-ml/opik/blob/ab0f8bb5074e4a12bde2b6a5af5ad161d28f1088/apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ProjectsResourceTest.java#L1358-L1361

Reply:
Can you rename it to something more significative and less confusing then? e.g: `avgTotalEstimatedCost` or similar.